test/include/pexcept_config.h

```diff
@@ -0,0 +1,32 @@
+#ifndef PUTILS_PEXCEPT_CONFIG_H
+#define PUTILS_PEXCEPT_CONFIG_H
+#endif
+
+#ifndef PEXCEPT_T
+#define PEXCEPT_T unsigned int
+#endif
+
+extern PEXCEPT_T uncaught_exception;
+extern short status;
+enum statuses {BEFORE_TRY, SUCCESS_TRY, AFTER_TRY, BEFORE_CATCH};
```

Comment:
Consider renaming this `enum` to its singular form, `status`, and possible add `_t` -> `status_t` . 

It usually confers a clear meaning when using the type in code, e.g: 

```c
int main(void) {
  enum statuses my_status = BEFORE_TRY;  //Note that the plural type name may hint that more than one status is possible at the same time
} 
```
vs

```c
int main(void) {
  enum status_t my_status = BEFORE_TRY; 
} 
```